WORKSPACE

```diff
@@ -16,11 +16,17 @@ go_repositories()
 
 go_repository(
     name = "com_google_cloud_go",
-    commit = "57377bad3486b37af17b47230a61603794c798ae",
+    commit = "3b1ae45394a234c385be014e9a488f2bb6eef821",
```

Comment:
Also, you need to update google.golang.org/api (https://code.googlesource.com/google-api-go-client/)